Structure_Engine/Query/Fixities.cs

```diff
@@ -38,7 +38,7 @@ public static partial class Query
         [Output("springValues", "The fixity values from a constraint as a bool array, where true indicates a fixity, in the following order: TransX, TransY, TransZ, RotX, RotY, RotZ.")]
         public static bool[] Fixities(this Constraint6DOF constraint)
         {
-            return new bool[] 
+            return constraint.IsNull() ? null : new bool[]
```

Comment:
As for the elastic values, but even trickier for this one due to null values for this class (i.e. defaults) would be different for BarRestraint and Nodal supports (bar restraint defaults being all fixed and node support defaults being all free).

For that reason I think it is reasonable to keep this one returning null, why it might be a good reason to keep the elastic values returning null as well, for symmetry.

Again, good to get this cleared and aligned, so glad for other peoples input. @peterjamesnugent @JosefTaylor @al-fisher @rolyhudson @FraserGreenroyd